src/node_env_var.cc

```diff
@@ -14,37 +14,64 @@ using v8::Array;
 using v8::Boolean;
 using v8::Context;
 using v8::EscapableHandleScope;
+using v8::HandleScope;
 using v8::Integer;
 using v8::Isolate;
+using v8::Just;
 using v8::Local;
+using v8::Maybe;
 using v8::MaybeLocal;
 using v8::Name;
 using v8::NamedPropertyHandlerConfiguration;
 using v8::NewStringType;
+using v8::Nothing;
 using v8::Object;
 using v8::ObjectTemplate;
 using v8::PropertyCallbackInfo;
 using v8::String;
 using v8::Value;
 
+class RealEnvStore final : public KVStore {
+ public:
+  Local<String> Get(Isolate* isolate, Local<String> key) const override;
+  void Set(Isolate* isolate, Local<String> key, Local<String> value) override;
+  int32_t Query(Isolate* isolate, Local<String> key) const override;
+  void Delete(Isolate* isolate, Local<String> key) override;
+  Local<Array> Enumerate(Isolate* isolate) const override;
+};
+
+class GenericKVStore final : public KVStore {
```

Comment:
`Generic` seems a bit..generic to me. Maybe `STLKVStore`? (wow that's more capital cases than I thought), or `MapKVStore`? Or something that indicates this is thread-safe? Or just some comments would be nice.

Reply:
Done! (Went with `MapKVStore`, I don’t think it’s important that this is backed by a particular STL container)